launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -13,6 +14,8 @@
   <arg name="all_camera_namespaces" default="[camera6, camera7]"/>
 
   <!-- ML parameters -->
+  <arg name="whole_image_detector_model_path" default="$(find-pkg-share autoware_tensorrt_yolox)/data"/>
+  <arg name="whole_image_detector_model_name" default="yolox_s_car_ped_tl_detector_960_960_batch_1.onnx"/>
```

Comment:
seems different from perception.launch.xml.
which one is right?

Reply:
resolved in https://github.com/autowarefoundation/autoware.universe/pull/9731/commits/21ad8bc1607eff4f97bd9aeb2c5fd5b4dea93598